### ISSUE TITLE
check-es-shard-allocation-status.rb & Elasticsearch 5.x

### ISSUE THREAD

**Issue by driegel** [sensu-plugins/sensu-plugins-elasticsearch#66]
Hello,

using check-es-shard-allocation-status.rb causes an error w/ Elasticsearch 5.x (I am using 5.1.2.):

```
check-es-shard-allocation-status.rb
Check failed to run: 400 Bad Request, ["/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/rest-client-1.8.0/lib/restclient/abstract_response.rb:74:in `return!'", "/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:495:in `process_result'", "/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:421:in `block in transmit'", "/opt/sensu/embedded/lib/ruby/2.3.0/net/http.rb:853:in `start'", "/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:413:in `transmit'", "/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:176:in `execute'", "/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:41:in `execute'", "/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/rest-client-1.8.0/lib/restclient/resource.rb:51:in `get'", "/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/sensu-plugins-elasticsearch-1.1.3/bin/check-es-shard-allocation-status.rb:57:in `get_es_resource'", "/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/sensu-plugins-elasticsearch-1.1.3/bin/check-es-shard-allocation-status.rb:65:in `master?'", "/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/sensu-plugins-elasticsearch-1.1.3/bin/check-es-shard-allocation-status.rb:82:in `run'", "/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/sensu-plugin-1.4.2/lib/sensu-plugin/cli.rb:58:in `block in <class:CLI>'"]
```

The reason is the url in line 65: 

> /_cluster/state?filter_routing_table=true&filter_metadata=true&filter_indices=true&filter_blocks=true&filter_nodes=true

Running this against Elasticsearch 5.x causes the following error:
```
GET _cluster/state?filter_routing_table=true&filter_metadata=true&filter_indices=true&filter_blocks=true&filter_nodes=true

{
  "error": {
    "root_cause": [
      {
        "type": "illegal_argument_exception",
        "reason": "request [/_cluster/state] contains unrecognized parameters: [filter_blocks] -> did you mean [filter_path]?, [filter_indices] -> did you mean [allow_no_indices]?, [filter_metadata] -> did you mean [filter_path]?, [filter_nodes] -> did you mean [filter_path]?, [filter_routing_table]"
      }
    ],
    "type": "illegal_argument_exception",
    "reason": "request [/_cluster/state] contains unrecognized parameters: [filter_blocks] -> did you mean [filter_path]?, [filter_indices] -> did you mean [allow_no_indices]?, [filter_metadata] -> did you mean [filter_path]?, [filter_nodes] -> did you mean [filter_path]?, [filter_routing_table]"
  },
  "status": 400
}
```

Thank you for looking into this.

**Comment by majormoses** [comment]
@driegel this should be fixed as of `1.2.0` can you verify?

**Comment by driegel** [comment]
@majormoses Thank you. It's fixed now.
